src/features/profile/edit-profile/modal-content.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Button, TextField } from "@mui/material";
+import { modalStyle } from "./styles/styles";
+import useEditProfileFormState from "./use-edit-profile-form-state";
+import useForm from "./use-form";
```

Comment:
Por el contrario aqui si creo hay que ser mas especifico, form que? de que?, la ruta si da un poco de contexto pero al ser un archivo diria ayudaria ser mas especifico
```suggestion
import useEditProfileForm from "./use-edit-profile-form";
```

Ademas de que se diferencia esto de `useEditProfileFormState`????

Si `useEditProfileFormState` se encarga de recopilar la informacion es un buen nombre o incluso Form sin STate

Pero entiendo que se encarga de definir si esta abierto o no el formulario??? siendo asi podria tener un nombre como 
`useOpenForm({ setIsOpen, user });`

O si es que se encarga de "enviar" el formulario entonces creeria seria mas bien como
`useDispatchEditProfile({ setIsOpen, user });`

o algo asi como
`useOnSubmitForm({ setIsOpen, user });`